REPO.bazel

```
@@ -0,0 +1 @@
+ignore_directories(["dist", "**/node_modules/**"])
```

Need to also include the `.angular` directory

```suggestion
ignore_directories([
  "dist",
  "**/node_modules/**",
  "**/.angular",
])
```